src/euphorie/content/browser/surveygroup.py

```diff
@@ -122,21 +124,43 @@ def buildSurveyTree(self):
         countries = [c for c in countries if c["sectors"]]
         return countries
 
+    def get_template_copy(self, container, template):
+        """Get's the copy for the template
+
+        The template is a survey that will be copied inside the newly
+        created survey group.
+        """
+        copy = template._getCopy(container)
+
+        # Reset the copied tree UIDs
+        # We use a fake event to reuse the subscriber from plone.uuid that
+        # resets the UIDs on copy
+        fake_copy_event = ObjectCopiedEvent(None, None)
+        addAttributeUUID(copy, fake_copy_event)
+        for id, item in copy.ZopeFind(copy, search_sub=1):
+            addAttributeUUID(item, fake_copy_event)
```

Comment:
I am a bit shy firing up the event because that can trigger unwanted subscribers

Reply:
I can imagine.  But I am a bit shy about *not* firing the event, because then we miss *wanted* subscribers.  I dug a bit and found a few that could be useful:

In `CMFCore`:

```
    elif IObjectCopiedEvent.providedBy(event):
        if hasattr(aq_base(ob), 'workflow_history'):
            del ob.workflow_history
```

In `plone.app.linkintegrity`:

```
    <subscriber
        for="plone.app.relationfield.interfaces.IDexterityHasRelations
             zope.lifecycleevent.interfaces.IObjectCopiedEvent"
        handler=".handlers.modifiedContent" />
```

In `plone.app.multilingual`:

```
@adapter(ITranslatable, IObjectCreatedEvent)
def addAttributeTG(obj, event):

    if (
        not IObjectCopiedEvent.providedBy(event)
        and getattr(aq_base(obj), ATTRIBUTE_NAME, None)
    ):
        return  # defensive: keep existing TG on non-copy create
```

Probably no big deal, but still my suggestion stands.

Anyway, this PR is an improvement already because it creates new UIDs, so I approve.